app/models/comment.rb

```diff
@@ -0,0 +1,7 @@
+class Comment < ApplicationRecord
+  belongs_to :post
+
+  belongs_to :post
+  belongs_to :author, class_name: 'User'
+  after_save :update_post_comments_counter
```

Comment:
- [ ] Great job using the Active Rails Callback - after_save, but I think you're missing the private method just as you did with the post model. Kindly add this private method for update_post_comments_counter

```
priavate  

def update_post__comments_counter
    author.increment!(:post_comments_counter)
  end

```

Reply:
@t-yanick i added it